var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -47,6 +47,11 @@ class Hdf5(CMakePackage):
 
     # Odd versions are considered experimental releases
     version("1.13.3", sha256="83c7c06671f975cee6944b0b217f95005faa55f79ea5532cf4ac268989866af4")
+    # NOTE: retain version 1.13.2 until:
+    # 1. hdf5-vol-async, hdf5-vol-cache, and hdf5-vol-log are ready to build off
+    #    hdf5@1.13.3+ (VOL struct underwent breaking changes).
```

Comment:
Probably should be "...ready to build off hdf5@1.14.0+

Reply:
All are ready except hdf5-vol-external-passthrough.  I don't think that one is of great importance to anyone outside of hdf5.  I can try to update it this afternoon, so you can wait for the update, or go ahead without it.  As I recall we kept hdf51.13.2 so e4s tests would pass - probably Paraview in particular.

Reply:
I don't think that gets built via any of the spack CI stacks, so it should be okay to follow-up with that.